src/targets/github.ts

```diff
@@ -473,8 +385,10 @@ export class GithubTarget extends BaseTarget {
 
     await Promise.all(
       localArtifacts.map(({ path, mimeType }) =>
-        this.uploadAsset(release, path, mimeType)
+        this.uploadAsset(draft, path, mimeType)
       )
     );
+
+    await this.publishReleaseDraft(draft.id);
```

Comment:
I think this method should accept a `release` object like `uploadAssets` for consistency. (or vice versa)

Reply:
Done in 7a968f67abeba6dadb462224bba5b1ed4f6fa1dc. 👍